Software/Development/SemesterProject/System/scheduler.c

```diff
@@ -23,10 +23,10 @@
 static enum taskListTag
 {
     task5msItem,
-    task10msItem,
     task20msItem,
     task200msItem,
     task1sItem,
+    task10msItem,
```

Comment:
Why did you change the position from the right place to the wrong place? C'mon Nicolai...